dqgen/services/additions_query_generator.py

```diff
@@ -6,25 +6,82 @@
 # Email: costezki.eugen@gmail.com 
 
 """ """
+from dqgen.adapters import template_builder, query_file_name_builder
 from dqgen.services.base_query_generator import BaseQueryGenerator
+from dqgen.services.query_template_registry import QueryTemplateRegistry
 
 
 class InstanceAdditionsGenerator(BaseQueryGenerator):
     """
-        TBD
+        This class will build the instance additions query as a file and
```

Comment:
Just realised that all these classes have the same body, but only one parameter is changing. We could refactor this and provide THAT as a constructor parameter, simplifying the code.